tests/test_export_dos_suppliers.py

```diff
@@ -0,0 +1,364 @@
+from itertools import chain
+
+import pytest
+import six
+from mock import Mock, call, patch, ANY, mock_open
+
+from dmscripts import export_dos_suppliers
+from dmscripts.insert_dos_framework_results import CORRECT_DECLARATION_RESPONSES
+from dmapiclient import HTTPError
+
+
+# TODO: move to conftest
```

Comment:
Is this TODO going to get done now, or some indeterminate time in the never future?